scripts/document-globals

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```

Comment:
For the script, let's:

1. Rename the script to `make-global-opts-documentation` to be consistent with the `make` prefix usage in the `scripts` directory.
2. We should make sure the file is executable (e.g. `chmod +x scripts/make-global-opts-documentation`) to be able to execute the file without needing to specify python.